lib/hsa/hsaPresumKernel.cpp

```diff
@@ -1,5 +1,13 @@
 #include "hsaPresumKernel.hpp"
 
+// Use old symbol naming convention if 
+// compiled with ROCM version 2.3 or older
+#ifdef USE_OLD_ROCM
```

Comment:
This will need to be done for every file.  Also it should be possible to simplify this to just add `.kd` or ` ` below and include it in the `hsaCommand.hpp` file.